src/bin/cargo/main.rs

```diff
@@ -47,6 +47,25 @@ fn main() {
     }
 }
 
+const BUILTIN_ALIASES: [(&str, &str); 4] = [
+    ("b", "alias: build"),
+    ("c", "alias: check"),
+    ("r", "alias: run"),
+    ("t", "alias: test"),
+];
+
+/// Function which contains the list of all of the builtin aliases and it's
+/// corresponding execs represented as &str.
+fn builtin_aliases_execs(cmd: &str) -> Option<&str> {
+    match cmd {
+        "b" => Some("build"),
```

Comment:
Instead of defining the set of aliases twice, perhaps just place them all in a single table?  Like a 3-element tuple `(alias, command, description)`, and then `builtin_aliases_execs` could just iterate over the array and find a match.

Reply:
That's brilliant!! Will modify it!!